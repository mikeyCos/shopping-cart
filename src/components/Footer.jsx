import "../styles/Footer.module.css";

const Footer = () => {
  return (
    <footer>
      <div>Placeholder</div>
    </footer>
  );
};

export default Footer;
