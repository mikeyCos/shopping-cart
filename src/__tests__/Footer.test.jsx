import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";
import Footer from "../components/Footer";

describe("Footer component", () => {
  it("The Footer component matches snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
