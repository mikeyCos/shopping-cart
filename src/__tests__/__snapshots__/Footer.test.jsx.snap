// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > The Footer component matches snapshot 1`] = `
<div>
  <footer>
    <div>
      Placeholder
    </div>
  </footer>
</div>
`;
